Chapters/2-2-Chapter9.md

```diff
@@ -0,0 +1,169 @@
+
+The woman pushed him ever closer toward the noise and slowly he gathered the nature of this place. The fire was a behemoth, eating through branches as thick as his head. It would have escaped and eaten the entire village had it not been dug into the ground and surrounded by thick firebrick. The forge and oven were cleverly situated mere steps away, with their own fires stoked by the much larger one. There were people cooking fish and kelp on the other side simply by putting them directly on the brick and flipping them over occasionally. The smell made Kipnu lose all his focus and he briefly considered stealing a morsel. Either through experience or observation, Setre instinctively said, "Don't even think about it."
+
+Of the nearly hundreds of people mulling about, each one had glassy eyes that glowed faintly blue, just like the kaln from the boat. He had seen patients like this before but never had he seen so many people under the power of kanosin in one place, and he wondered if they were kept like this on purpose; clearly his new friend was different.
+
+"Who are these people?" he asked.
+
+"The working class. They come here for the fix and work to earn it."
+
+"What do they do after their sentence is up?"
+
+"Most of them never leave."
+
+Kipnu was incredulous, but was in no position to change a mind. He knew that the murmur of people had to be some other sort of group.
+
+"What about the others?"
+
+"You'll see."
+
+At the far side of the fire a giant yurt stood proud, painted in long strokes of ochre on the same thick kelp that made up the woman's coat. Kipnu could hear the noises of a nearly a thousand people inside of it.
+
+The woman took his shoulder and said, "In here, you stay close to me."
+
+He nodded. He didn't really have a choice.
+
+She moved to a fissure in the kelpen wall and glanced into the yurt, then turned her head back to him. "Before we go in, what is your name and class?"
+
+"Kipnu. But Heiko doesn't have classes. I'm a doctor if that's what you mean... Why do you need to know this?"
+
+She looked him up and down again as if her eyes had seen him for the first time.
+
+"I need to call you something when I introduce you."
+
+The two of them ducked under the flaps of the wall and entered a new world of blue. Everyone was nude or nearly so, and had ornately-patterned luminescent markings on their bodies, much like this woman's. A single drum pattern rattled the ground while musicians played their melodies and the dancers blurred into the sweet smelling smoke of anomin oil, their skin filling the air with an otherworldly glow. And still others were in the throws of erotic ecstasy, divining the truth in each other. Their eyes were brightly-colored with the same light, and he knew that they too were under the spell of kanosin.
```

Comment:
So kanosin is like an ecstasy drug? In earth human terms, what neurotransmitters does it activate?
One pet peeve that sometimes in fictional accounts of altered state of consciousness experiences mixes how people react to include every crazy thing under the sun and moon. It makes it hard to suspend disbelief. You can't be relaxed and excited at the same time.